packages/connectors/src/MediaConnector.ts

```diff
@@ -42,4 +42,5 @@ export type MediaConnectorCapabilities = {
     query: boolean;
     detail: boolean;
     filtering: boolean;
+    metadata?: boolean;
```

Comment:
Is this nullable to make it backwards compatible?

Reply:
Yes

Reply:
This will not work without engine changes, as these capabilities are typed so they will get lost.

Reply:
@Matthiee can we make sure to fix this? connectors are public for a month or so now so we basically can't do breaking changes anymore